api/migrations/20210427172703-legislationToArray.js

```diff
@@ -73,23 +53,28 @@ exports.up = async function(db) {
               legislationDescription: ""
             }
           }
-        );
+        ).then( result => {
+          if (result.modifiedCount > 0 ) {
+            numUpdatedNrpti++;
+          }
+        });
```

Comment:
Don't mix async/await and .then.  Better to do something like:

```
const result = await redacted_record_subset.updateOne(...
if (result.modifiedCount > 0) {
  numUpdatedNrpti++;
}
...
```

Since the await "waits" for the execution to finish before moving onto the next line, the return value is the payload of the function call so you can just assign the return value into an object and check it on the next line.

Reply:
nice catch tyty